src/apps/btc/btc_common.h

```diff
@@ -253,6 +254,7 @@ USE_RESULT bool btc_common_multisig_is_valid(
 USE_RESULT bool btc_common_multisig_hash(
     BTCCoin coin,
     const BTCScriptConfig_Multisig* multisig,
+    bool sort_xpubs,
```

Comment:
I would've added 2 extra functions, `btc_common_multisig_hash_sorted` and `btc_common_multisig_hash_unsorted` and converted `btc_common_multisig_hash` to a static `_btc_common_multisig_hash`. They would both simply call `_btc_common_multisig_hash` with either `true` or `false`.

It just makes it explicit and more clear to see what's happening in the rest of the code. For example, compare

```c
    if (!btc_common_multisig_hash(coin, multisig, false, keypath, keypath_len, hash)) {
        return APP_BTC_ERR_UNKNOWN;
    }
```

to 

```c
    if (!btc_common_multisig_hash_unsorted(coin, multisig, keypath, keypath_len, hash)) {
        return APP_BTC_ERR_UNKNOWN;
    }
```

assuming you don't remember what the signature of the function is and without looking it up.

Reply:
what about `bool sorted = false; btc_common_multisig_hash(coin, multisig, sorted, keypath, keypath_len, hash)`?

Reply:
yeah, that would also work, I guess. personally, I'd still use 2 separate functions: just feels eaier to read and harder to make a silly mistake setting the variable to incorrect value or be somehow theorically flipped true/false intentionally or otherwise because variables are mutable whereas functions and the names they point to in a read-only executable memory are not.

but whichever you like best.

Reply:
Added the two functions :+1: